nvflare/lighter/impl/static_file.py

```diff
@@ -21,6 +21,20 @@
 
 class StaticFileBuilder(Builder):
     def __init__(self, enable_byoc=False, config_folder="", app_validator="", docker_image=""):
+        """Uses the information from project.yml through study to go through the participants and write the contents of
+        each file with the template, and replacing with the appropriate values from project.yml.
+
+        Usually, two main categories of files are created in all FL participants, static and dynamic. Static files
+        have similar contents among different participants, with small differences.  For example, the differences in
+        sub_start.sh are client name and python module.  Those are basically static files.  This builder uses template
+        file and string replacement to generate those static files for each participant.
```

Comment:
Do we have an explanation of a dynamic file?
Like a builder or something, or other than static file are all dynamic files?

Reply:
Static files are derivatives from templates.  Dynamic files are usually generated by other builders, with their 'build' methods.  For example, CertBuilder writes all certs during build.  Those certs are dynamic files.